spec/services/persist_directly_contained_output_file_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe CurationConcerns::PersistDirectlyContainedOutputFileService do
+  let(:object)            { FileSet.create! { |fs| fs.apply_depositor_metadata('justin') } }
```

Comment:
Rubocop doesn't care about spaces for padding.  This is copying the style of the existing test for the BasicContainedOutputFileService.